CHANGELOG.md

```diff
@@ -1,5 +1,188 @@
+
+    [ashleymical]: https://github.com/ashleymichal
+    [issue/219]: https://github.com/cloudflare/wrangler/issues/219
+    [pull/263]: https://github.com/cloudflare/wrangler/pull/263
+
+  - **Adds more descriptive subdomain errors - [EverlastingBugstopper], [issue/207] [pull/259]**
+
+    Warns users when attempting to add a subdomain w/o account_id in wrangler.toml
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/207]: https://github.com/cloudflare/wrangler/issue/207
+    [pull/259]: https://github.com/cloudflare/wrangler/pull/259
+
+  - **Allow custom webpack configuration in wrangler.toml - [EverlastingBugstopper], [issue/246] [pull/253]**
+
+    Users can now specify custom webpack configuration filename, via a `webpack_config` key in `wrangler.toml`.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/246]: https://github.com/cloudflare/wrangler/issue/246
+    [pull/253]: https://github.com/cloudflare/wrangler/pull/253
+
+  - **Enforce one webpack config - [xtuc], [pull/245]**
+
+    We don't support multiple webpack configurations, for better UX we should make sure to enforce it.
+
+    [xtuc]: https://github.com/xtuc
+    [pull/245]: https://github.com/cloudflare/wrangler/pull/245
+
+  - **Add user agent - [xtuc], [issue/234] [pull/236]**
+
+    Set the user-agent for every request: `wrangler/dev` in debug mode and `wrangler/$version` in release mode.
+
+    [xtuc]: https://github.com/xtuc
+    [issue/234]: https://github.com/cloudflare/wrangler/issue/234
+    [pull/236]: https://github.com/cloudflare/wrangler/pull/236
+
+  - **Display commands in their defined order - [Electroid], [pull/236]**
+
+    Clap was re-arranging commands alphabetically, this change lets us define our own order. This moves help to the bottom so it's less awkward (since it has no emoji!) Also fixed some emoji spacing issues.
+
+    [Electroid]: https://github.com/Electroid 
+    [pull/233]: https://github.com/cloudflare/wrangler/pull/236
+
+  - **Show sizes - [xtuc], [pull/205]**
+
+    Once the build is finished print the size of the script, in gzip, and Wasm binary size if available.
+
+    [xtuc]: https://github.com/xtuc
+    [pull/205]: https://github.com/cloudflare/wrangler/pull/205
+
+  - **ux: add HTTP prefix to publish command output. - [elithrar], [pull/198]**
+
+    Prefix "https://" in front of the "script available" output to allow shells to automatically detect it as a link.
+
+    [elithrar]: https://github.com/elithrar
+    [pull/198]: https://github.com/cloudflare/wrangler/pull/198
+
+  - **Fix installer - [xtuc], [pull/195]**
+
+    [xtuc]: https://github.com/xtuc
+    [pull/195]: https://github.com/cloudflare/wrangler/pull/195
+
+  - **Build: add message and emoji - [xtuc], [pull/193]**
+
+    [xtuc]: https://github.com/xtuc
+    [pull/193]: https://github.com/cloudflare/wrangler/pull/193
+
+- ### 🤕 Fixes
```

Comment:
for all of these- we should explain the bug.. what the user would see.. a brief explanation of why that was happening, and then how we fixed it